elasticsearch/client/__init__.py

```diff
@@ -1164,7 +1164,8 @@ def bulk(self, body, index=None, doc_type=None, params=None):
         if body in SKIP_IN_PATH:
             raise ValueError("Empty value passed for a required argument 'body'.")
         return self.transport.perform_request('POST', _make_path(index,
-            doc_type, '_bulk'), params=params, body=self._bulk_body(body))
+            doc_type, '_bulk'), params=params, body=self._bulk_body(body),
+            headers={'Content-Type': 'application/x-ndjson'})
```

Comment:
we use lowercase for all headers to avoid conflicts - see https://github.com/elastic/elasticsearch-py/blob/master/elasticsearch/connection/http_urllib3.py#L59-L62

A test is needed to make sure the headers are merged properly